src/app/api/contributor/schema.ts

```diff
@@ -0,0 +1,18 @@
+import { ContributionType } from "@prisma/client";
+import { z } from "zod";
+
+export const getContributionSchema = z.object({
```

Comment:
is this used anywhere?

Reply:
I am not too sure because I think Kristin wrote this, and I kinda just left it there in case that it is used for code somewhere else. I can delete if you want

Reply:
deleted